src/ethereum/info/pulsechainInfo.ts

```diff
@@ -0,0 +1,106 @@
+import { EdgeCurrencyInfo, EdgeTokenMap } from 'edge-core-js/types'
+
+import { makeOuterPlugin } from '../../common/innerPlugin'
+import { makeMetaTokens } from '../../common/tokenHelpers'
+import type { EthereumTools } from '../EthereumTools'
+import type { EthereumFees, EthereumNetworkInfo } from '../ethereumTypes'
+
+const builtinTokens: EdgeTokenMap = {}
```

Comment:
We should allow users to add custom tokens and to do that we need to add at least one builtin token. Maybe add wrapped PLS https://scan.pulsechain.com/address/0xA1077a294dDE1B09bB078844df40758a5D0f9a27